lib/main.dart

```diff
@@ -16,6 +20,16 @@ Future<void> main() async {
 class MyApp extends StatelessWidget {
   @override
   Widget build(BuildContext context) {
+    return MaterialApp(
```

Comment:
No requirement for this return statement, we already have one below. Remove this one and add the route in the below one.